frontend/projects/src/app/components/form/TextBox/TextBox.tsx

```diff
@@ -0,0 +1,26 @@
+import { UseFormReturn } from "react-hook-form";
+import type { FieldValues, Path } from "react-hook-form";
+import style from "./TextBox.module.css";
+
+type TextBoxProps<T extends FieldValues> = {
+    title: string;
+    name: Path<T>;
+    form: UseFormReturn;
```

Comment:
```suggestion
    form: UseFormReturn<T>;
```
ここも修正していただけると幸いです
多分これつけないと型エラーが出ると思います

Reply:
見逃してました
ありがとござます